TimoCloud-API/src/main/java/cloud/timo/TimoCloud/api/objects/properties/BaseProperties.java

```diff
@@ -13,18 +12,30 @@
     private Integer maxRam;
     private Integer keepFreeRam;
     private Double maxCpuLoad;
+    private String publicIpConfig;
 
     private BaseProperties() {
         this.maxRam = getDefaultPropertiesProvider().getMaxRam();
         this.keepFreeRam = getDefaultPropertiesProvider().getKeepFreeRam();
         this.maxCpuLoad = getDefaultPropertiesProvider().getMaxCpuLoad();
+        this.publicIpConfig = "AUTO";
     }
 
     public BaseProperties(String id, String name, PublicKey publicKey) {
         this();
         this.id = id;
         this.name = name;
         this.publicKey = publicKey;
+        this.publicIpConfig = "AUTO";
```

Comment:
Please add an extra constructor accepting a field "publicIpConfig" and change this one to
```
public BaseProperties(String id, String name, PublicKey publicKey) {
    this(id, name, publicKey, getDefaultPropertiesProvider().getPublicIpConfig());
}
```